### ISSUE TITLE
Frontend improvements

### ISSUE THREAD

**Issue by LaSav** [mnm-matin/ai_alignment_graph#15]
- UX prompt writing

**Comment by mnm-matin** [comment]
@LaSav just had a play around with the [quartz layout config](https://github.com/mnm-matin/ai_alignment_graph/blob/v4/quartz.layout.ts), one could move `Component.Search()` from the left layout to the center layout quite easily by moving it to the pageLayout instead.

**Comment by LaSav** [comment]
@mnm-matin Yep, that will do the trick, however that will apply the layout for all pages.

I think we should create a separate landing page component with a custom layout, and flag the component to render in the `renderPage.tsx` like in this [example](https://github.com/CatCodeMe/quartz_local/commit/dd525936deec9032a9cf1840933c96abed760dea).

Other methods to implement a custom landing page can be found in this Quartz discord [chat](https://discord.com/channels/927628110009098281/1211952698673602580) but the one from above stood out to me the most.

**Comment by mnm-matin** [comment]
@LaSav sounds like a plan! Feel free to create the PR.
